godockerize.go

```diff
@@ -184,7 +188,10 @@ func doBuild(c *cli.Context) error {
 
 	for _, importPath := range packages {
 		fmt.Printf("godockerize: Building Go binary %s...\n", path.Base(importPath))
-		cmd := exec.Command("go", "build", "-buildmode", "exe", "-tags", "dist", "-o", path.Base(importPath), importPath)
+		flags := c.StringSlice("go-build-flags")
```

Comment:
doesn't seem like you need or benefit much from this variable. I would just inline it below.